Code/MinimalLib/jswrapper.cpp

```diff
@@ -553,29 +552,44 @@ EMSCRIPTEN_BINDINGS(RDKit_minimal) {
       .function("straighten_depiction",
                 select_overload<void()>(&JSMol::straighten_depiction))
       .function("straighten_depiction",
-                select_overload<void(bool)>(&JSMol::straighten_depiction));
+                select_overload<void(bool)>(&JSMol::straighten_depiction))
+      .function("get_num_atoms", select_overload<unsigned int(bool) const>(
+                                     &JSMol::get_num_atoms))
+      .function("get_num_atoms",
+                select_overload<unsigned int() const>(&JSMol::get_num_atoms))
+      .function("get_num_bonds", &JSMol::get_num_bonds)
+      .function("get_copy", &JSMol::get_copy, allow_raw_pointers());
```

Comment:
What's this? Is it related to the PR topic?

Reply:
`get_num_atoms()` and `get_num_bonds()` are related (see above). `get_copy()` is not and I have removed it.